lambda/update_security_groups.py

```diff
@@ -0,0 +1,166 @@
+import boto3
```

Comment:
Do you have any thoughts on how we can keep this file up-to-date? It's a shame it's not already a library. We should probably at least add a comment explaining where it came from, so it's easy to up date it manually.

We should add the original code licence too.

Reply:
Again, there's nothing to really keep up to date. If it becomes a problem, we can think about it then but there's no need to now.